pages/chapters/index.tsx

```diff
@@ -19,7 +19,8 @@ const ChapterPage: NextPage<Props> = ({ chapter }) => {
     return (
         <div>
             <Head>
-                <title>Chapters | MindVersity | A peer mental health network.</title>
+                <title>MindVersity Chapters | MindVersity - A peer mental health network.</title>
```

Comment:
This one's title can be `Chapters | MindVersity...` instead of `MindVersity Chapters | MindVersity...`. I think having MindVersity twice is a bit repetitive.